### PR TITLE
ETS-2157 - removes js calculated position of Skyscraper

### PR DESCRIPTION
**Pull request by virgilchiriac**

### CODE REVIEW

**Review by BirgitPohl — APPROVED** [review]
👍🏻Awesome. It's a much easier solution. Thank you for contributing to it.

**Review by dknx01 — DISMISSED** [review]

**Review by susannegernandt — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+## 0.43.5
```
Comment (BirgitPohl) [inline CHANGELOG.md]: Since Claudia merged her PR, you need to update your version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
...
+- Removes js positioning of Skyscraper
```
Comment (BirgitPohl) [inline CHANGELOG.md:6]: Since Claudia merged her PR, you need to update your version.

**Review by BirgitPohl — CHANGES_REQUESTED** [review]
Also, when merging, try to squash the commits before. Thank you. There is the option to squash merge on Github.

**Review by BirgitPohl — APPROVED** [review]

**Pull request merged**